### DISCUSSION TITLE
Fish Autocomplete and contributing

### DISCUSSION THREAD

**Discussion by mindoodoo** [ajeetdsouza/zoxide#399] [General]
Hey,
Just came across  zoxide recently while looking for modern replacements for all the classic commands I use every day.
The project is and runs like a charm except that I don't seem to have tab autocomplete in fish.
Having said that, I would love to look into contributing that functionality to the project however having found no contributors guide I thought it best to discuss it here first.

**Comment by ajeetdsouza** [comment]
- What version of zoxide are you on? (`zoxide --version`)
- Is this the same problem as https://github.com/ajeetdsouza/zoxide/issues/341?
